mitodl/open-discussions
# add comment sorting UI

#### What are the relevant tickets?

closes #229 

#### What's this PR do?

this adds some UI to set the 'sort' param when you're fetching the comments for a post. Basically we add a little dropdown that has the sorting options, and we you pick one we re-issue the request for comments to the backend with the relevant param.

#### How should this be manually tested?

You should be able to load comments and posts normally, and you should be able to make comments, and all that should work as before.

The comment sort dropdown should have a default value of 'Best'. Selecting 'New' or 'Old' should re-fetch the comments, and the sort method should be passed to the backend as a URL query param.

I found that the 'Old' sort often changed the order from 'Best' and 'New', so switching between those two on a post with a lot of comments can be a way to confirm that the order is changing.
